omim2obo/main.py

```diff
@@ -231,18 +243,53 @@ def run(use_cache: bool = False):
-        retrieve_mim_file('morbidmap.txt', download_files_tf))
-    for mim_number in morbid_map:
-        phenotype_mim_number, cyto_location = morbid_map[mim_number]
-        if phenotype_mim_number:
-            # RO:0003303 == 'causes condition' https://www.ebi.ac.uk/ols/ontologies/ro/properties?iri=http://purl.obolibrary.org/obo/RO_0003303
-            graph.add((OMIM[mim_number], RO['0003303'], OMIM[phenotype_mim_number]))
+    # todo: when addressed, remove this from the issue report JSON (or remove the report entirely)
+    morbid_map: Dict = parse_morbid_map(get_mim_file('morbidmap.txt', download_files_tf))
+    for mim_number, mim_data in morbid_map.items():
+        # todo?: unused `mim_data` keys:
+        #   'gene_mim_number': Should this be used somewhere?
+        #   'gene_symbols':  Should this be used somewhere?
+        #   'phenotype_label': I think this already added to graph because the MIM# and label shared in other file(s)
+        cyto_location: str = mim_data['cyto_location']
         if cyto_location:
             # What's 9606chr - joeflack4 2021/11/11
             chr_id = '9606chr' + cyto_location
-            # What's RO['0002525'] - joeflack4 2021/11/11
+            # RO:0002525 (is subsequence of)
+            # https://www.ebi.ac.uk/ols/ontologies/ro/properties?iri=http://purl.obolibrary.org/obo/RO_0002525
             graph.add((OMIM[mim_number], RO['0002525'], CHR[chr_id]))
+        assocs: List[Dict] = mim_data['phenotype_associations']
+        for assoc in assocs:
+            # RO:0003303 (causes condition)
+            # https://www.ebi.ac.uk/ols/ontologies/ro/properties?iri=http://purl.obolibrary.org/obo/RO_0003303
+            p_mim, p_lab, p_map_key, p_map_lab = \
+                assoc['phenotype_mim_number'], assoc['phenotype_label'], assoc['phenotype_mapping_info_key'], \
+                assoc['phenotype_mapping_info_label']
+            # i. Add to MIM
+            b = BNode()
+            graph.add((b, RDF['type'], OWL['Restriction']))
+            graph.add((b, OWL['onProperty'], RO['0003303']))
+            if p_mim:
+                graph.add((b, OWL['someValuesFrom'], OMIM[p_mim]))
+            else:
+                graph.add((b, OWL['someValuesFrom'], Literal(p_lab)))
+            graph.add((OMIM[mim_number], RDFS['subClassOf'], b))
+
+            # ii. add axiom
+            b2 = BNode()
+            graph.add((b2, RDF['type'], OWL['Axiom']))
+            graph.add((b2, OWL['annotatedSource'], OMIM[mim_number]))
+            graph.add((b2, OWL['annotatedProperty'], RDFS['subClassOf']))
+            # todo: add biolink evidence/genediseaseassoc?
+            # todo: add mapping key and related info in a different way?
+            graph.add((b2, RDFS['comment'], Literal(f'Evidence: ({p_map_key}) {p_map_lab}')))
+            b3 = BNode()
+            graph.add((b3, RDF['type'], OWL['Restriction']))
+            graph.add((b3, OWL['onProperty'], RO['0003303']))
+            if p_mim:
```

Comment:
Dthis dpes not work correctly:

```
OMIM:101850 a owl:Class ;
    rdfs:label "palmoplantar keratoderma, punctate  iia 3" ;
    IAO:0000142 PMID:13116681,
        PMID:141935,
        PMID:4449867,
        PMID:4694520,
        PMID:6221990,
        PMID:8651714,
        PMID:9870680 ;
    RO:0002525 CHR:9606chr2p25-p12 ;
    oboInOwl:hasExactSynonym "PPKP3",
        "acrokeratoelastoidosis",
        "collagenous plaques of hands and feet",
        "palmoplantar keratoderma, punctate  iia 3" ;
    rdfs:subClassOf [ a owl:Restriction ;
            owl:onProperty RO:0003303 ;
            owl:someValuesFrom "?Keratoderma, palmoplantar, punctate type 3" ] ;
    skos:exactMatch UMLS:C0545044 ;
    biolink:category biolink:Disease .
```

Reply:
@matentzn I should have brought this up as a question to you today. 

`?Keratoderma, palmoplantar, punctate type 3` is one of those `Phenotype` in `morbidmap.txt` that does not have a MIM number. After discussion on Friday Nov 11, our decision was to add the relationship to the label, rather than the MIM number, that's what you see here. However, after our discussion today, Nov 18, we've learned that these rows with no MIM # are problematic parts of the data model. Exomiser skips. We will also skip for now until we know what to do with them.

So, I will update the code to not add these relationships / reifications in these cases.

Reply:
`omim-temp.ttl`. Slipped in; will delete! Thanks for noticing.

Reply:
`omim-temp.ttl` deleted, and the "something gone wrong" (as described above) are complete, so marking resolved.